spec/tools/custom_cops/dont_include_sidekiq_worker_spec.rb

```diff
@@ -2,19 +2,15 @@
 require 'rubocop/rspec/support'
 require Rails.root.join('tools/custom_cops/dont_include_sidekiq_worker.rb')
 
-RSpec.describe CustomCops::DontIncludeSidekiqWorker do
+RSpec.describe CustomCops::DontIncludeSidekiqWorker, :config do
```

Comment:
(I simplified this spec slightly, because I was then going to use it as a basis for the new cop spec added in this PR, so I wanted to simplify it as much as reasonably possible, first.)